### PR TITLE
Collijk/feature/postprocess hospital

### PR DESCRIPTION
**Pull request by collijk**
Thread hospital data and new extra outputs through postprocessing.

This also cleans up some inconsistencies in the command templates.

### CODE REVIEW

File: specification_templates/postprocessing.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
     - 'worse'
     - 'reference'
-    - 'no_vaccine'
-    - 'vaccine_fast'
-    - 'vaccine_fast_elderly'
-    - 'best_masks'
+#    - 'no_vaccine'
+#    - 'vaccine_fast'
+#    - 'vaccine_fast_elderly'
+#    - 'best_masks'
```
Comment (miker985) [inline specification_templates/postprocessing.yaml:9]: should these be deleted?

**Review by kathrynline — APPROVED** [review]

File: src/covid_model_seiir_pipeline/pipeline/postprocessing/model/final_outputs.py [hunk 2]
```diff
@@ -73,11 +73,63 @@ def __init__(self,
         cumulative_label='cumulative_infections',
         aggregator=aggregators.sum_aggregator,
     ),
+    'r_controlled': MeasureConfig(
+        loaders.load_r_controlled,
+        'r_controlled',
+        splice=False,  # TODO: Get a few rounds of this and then set splice to True
```
Comment (miker985) [inline src/covid_model_seiir_pipeline/pipeline/postprocessing/model/final_outputs.py:79]: if all these `splice` args are the same thing then you should set a single variable and reuse it.

**Review by miker985 — APPROVED** [review]

File: src/covid_model_seiir_pipeline/pipeline/postprocessing/model/final_outputs.py (hunk 2 above)
Comment (collijk) [reply src/covid_model_seiir_pipeline/pipeline/postprocessing/model/final_outputs.py:79]: good call

File: specification_templates/postprocessing.yaml (hunk 1 above)
Comment (collijk) [reply specification_templates/postprocessing.yaml:9]: You should ignore churn in the specifications.  I'll set up a gitignore for the ones I use for testing soon.

**Pull request merged**